openapi/multi-yaml/index.yaml

```diff
@@ -28,7 +28,7 @@ tags:
       You can retrieve information about your payments and refund them if needed.
   - name: Payment Intents (use checkouts)
     description: |
-      **Please use the [Checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts) directly. instead of payment intents. This API will be deprecated in the future and replaced with the checkout API.**
+      **We no longer allow new platforms to use this API. Use the [Checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts) instead of payment intents. This API will be retired in the future as existing integrators to migrate.**
```

Comment:
I'd change the second sentence to: `Use the [Checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts) instead.`
The last sentence needs a slight tweak: `... as existing integrators migrate.`